api/src/test/java/org/openmrs/module/patientqueueing/api/PatientQueueingServiceTest.java

```diff
@@ -158,4 +162,41 @@ public void savePatientQueue_shouldCompleteExistingPatientQueueWithSamePatientAn
 		Assert.assertEquals(PatientQueue.Status.COMPLETED, patientQueue1.getStatus());
 		
 	}
+	
+	@Test
+	public void generateQueueNumber_shouldReturnNewQueueNumberForGivenPatientOnGivenDate() {
+		PatientQueueingService patientQueueingService = Context.getService(PatientQueueingService.class);
+		
+		Patient patient = Context.getPatientService().getPatient(10000);
+		
+		Location location = Context.getLocationService().getLocation(1);
+		
+		String queueNumber = patientQueueingService.generateQueueNumber(location, new Date(), patient);
+		
+		Assert.assertEquals(QueueingUtils.convertDateToDefaultDateFormatAsString(new Date(), null) + "-Unk" + "-001", queueNumber);
+		
+	}
+	
+	@Test
+	public void generateQueueNumber_shouldReturnExistingQueueNumberForGivenPatientOnGivenDate() {
+		PatientQueueingService patientQueueingService = Context.getService(PatientQueueingService.class);
+		String sDate1 = "2019-10-07 18:53:56";
+		Date date = null;
+		
+		try {
+			date = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").parse(sDate1);
+		}
+		catch (ParseException e) {
+			e.printStackTrace();
```

Comment:
https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotprintout

Reply:
Why are you even catching the exception?